File: server/GarageAPI/Models/ParkingLog.cs
using Core;
using GarageAPI.Enums;
using System;

namespace GarageAPI.Models
{
    public class ParkingLog : BaseEntity
    {
        public Guid CardId { get; set; }
        public Guid ParkingLevelId { get; set; }
        public Guid ParkingSpaceId { get; set; }
        public int ParkingSlot { get; set; }
        public string LicensePlate { get; set; }
        public VehicalType VehicalType { get; set; }
        public long ParkTime { get; set; }
        public long? LeaveTime { get; set; }
    }
}

File: client/src/app/modules/garage/classes/parking-log.ts
import { VehicalType } from "@modules/garage/enums/vehical-type";

export class ParkingLog {
  cardId: string;
  parkingLevelId: string;
  parkingSpaceId: string;
  parkingSlot: string;
  licensePlate: string;
  vehicalType: VehicalType
  parkTime: number;
  leaveTime: number;
}
